feconf.py

```diff
@@ -331,7 +331,9 @@ def get_empty_ratings():
 }
 
 # TODO(sll): Add all other URLs here.
-COLLECTION_DATA_URL_PREFIX = '/collectionhandler/data'
+COLLECTION_DATA_URL_PREFIX = '/collection_handler/data'
+COLLECTION_WRITABLE_DATA_URL_PREFIX = '/collection_editor/data'
```

Comment:
Should this be collection_editor_handler/data (per convention to use 'handler' for all json data urls)?